WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -208,6 +208,10 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonProperty(PropertyName = "AffiliateServers", DefaultValueHandling = DefaultValueHandling.Populate)]
 	public ImmutableDictionary<string, string> AffiliateServers { get; set; } = ImmutableDictionary<string, string>.Empty;
 
+	[DefaultValue(true)]
+	[JsonProperty(PropertyName = "DelayTransactionSigning", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public bool DelayTransactionSigning { get; set; } = false;
```

Comment:
Could be a TimeSpan as well like phase timeouts. Also, it can be a minimum delay that all the clients will wait. By default, it is zero => turned off (current behavior). On top of the minimum, the random Poisson will be added. How long does it take for the Trezor device to sign?

Reply:
>  How long does it take for the Trezor device to sign?

It depends on the size of the transaction but in general it needs tens of seconds to sign.